packages/mux-player-react/src/lazy.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import Suspense from './BrowserOnlySuspense';
+import { MuxPlayerProps, MuxPlayerRefAttributes } from './index';
+
+const MuxPlayerIndex = React.lazy(() => import('./index'));
+
+const Fallback = (props: Omit<MuxPlayerProps, 'playerSoftwareVersion' | 'playerSoftwareName'>) => {
+  const { style, ...rest } = props;
+  return (
+    <>
+      {/* TODO: can we add mux-player to JSX.IntrinsicElements */}
+      {/* @ts-ignore */}
+      <mux-player
+        data-mux-player-react-placeholder
+        style={{
+          // default mux-player styling
+          aspectRatio: '16/9',
+          display: 'block',
+          backgroundColor: '#000',
+          width: '100%',
+          // some styles specific to this placeholder
+          position: 'relative',
+          backgroundImage: props.placeholder ? `url(${props.placeholder})` : undefined,
+          backgroundRepeat: 'no-repeat',
+          backgroundSize: 'contain',
+          // user-implemented styles
+          ...style,
+        }}
+        // we pass the rest of the props to mux-player too.
+        // There is a possibility that mux-player imports before Suspense has had the opportunity
+        // to replace this placeholder with MuxPlayerReact.
+        // In that case, we want this placeholder to look as much like the incoming player as possible.
+        {...rest}
```

Comment:
I'm _sure_ this has side-effects that I haven't thought of yet.

Reply:
> since the bundle is already imported, the fallback will be acting like a normal mux player until isIntersecting notifies React to replace the fallback with the real MuxPlayerReact

I see. That makes perfect sense. So the clever hack of putting an un-registered custom element of `<mux-player>` as the placeholder during suspense is coming back to haunt us

Reply:
We _could_ fix this with a wrapper.... though then we'd have to add additional/different styling guidelines for /lazy. It won't be just a drop-in replacement.

Reply:
Oh btw it's not just the risk of multiple interactions. We also have this multiple-loading flicker.

https://user-images.githubusercontent.com/8933386/194163231-b12f8f83-fb3d-4f75-95e2-e0ef76f1c212.mov

Reply:
Mmkay I went for the wrapper approach. Feels like an API defeat, but I'm just too worried about jank from our original approach.